wordpress-mobile/WordPress-iOS
# feat: Retry all failed uploads on manual retry

## Description

Retrying all failed media uploads when tapping the "Retry" button for a
single media item improves the UX and aligns with the Android product's
existing behavior.

Fixes https://github.com/wordpress-mobile/gutenberg-mobile/issues/6408.

## Testing Instructions
1. Disable network connectivity for your device.
1. Launch editor.
1. Add a Gallery block.
1. Add two or more images to the block.
1. Enable network connectivity for your device.
1. Tap one of the images.
1. Tap "Retry."
1. Verify all images retry uploads.

## Regression Notes
1. Potential unintended areas of impact
    Media upload for various block types.
2. What I did to test those areas of impact (or what existing automated tests I relied on)
    Inserted and attached media for various block types.
3. What automated tests I added (or what prevented me from doing so)
    None, lack of familiarity with testing technology and lack of example unit
tests.

PR submission checklist:


UI Changes testing checklist:
